src/startApp.ts

```diff
@@ -63,7 +65,16 @@ async function loadEntryPointFile(functionAppDirectory: string): Promise<void> {
         try {
             const files = await globby(entryPointPattern, { cwd: functionAppDirectory });
             if (files.length === 0) {
-                throw new AzFuncSystemError(`Found zero files matching the supplied pattern`);
+                let message: string = globby.hasMagic(entryPointPattern, { cwd: functionAppDirectory })
+                    ? 'Found zero files matching the supplied pattern'
+                    : 'File does not exist';
+
+                if (entryPointPattern === 'index.js') {
+                    // This is by far the most common error and typically happens by accident, so we'll give these folks a little more help
+                    message += '. Learn more here: https://aka.ms/AAla7et';
```

Comment:
Can we have a more descriptive aka.ms link?

Reply:
The reason I use aka.ms links is for their ability to change the underlying url at any time (without needing to ship the worker), and because it's way shorter than the real url. I've got about 40 urls I've created over the years. Ain't nobody got time to come up with cute names for all of them, plus you wanna save the good names for when you actually need them (like big announcements or social media or whatnot)